bytestring/common/src/ByteString.kt

```diff
@@ -69,6 +70,7 @@ public class ByteString private constructor(
     public constructor(data: ByteArray, startIndex: Int = 0, endIndex: Int = data.size) :
             this(data.copyOfRange(startIndex, endIndex), null)
 
+    @Volatile
```

Comment:
Does it make sense to add an expect annotation like `VolatileInNative` and actualize it with `Volatile` only in native?

Like in [this patch](https://gist.github.com/fzhinkin/5cd4a5af93c6c6611ed7d8f405a55fa7)

Reply:
The nicest alternative would be to somehow perform load/store w/ relaxed memory ordering, though:
https://godbolt.org/z/rEdcobsW3